src/core/Akka.Streams/Dsl/GraphDsl.cs

```diff
@@ -471,10 +471,11 @@ public static class ForwardOps
         /// <typeparam name="TIn">TBD</typeparam>
         /// <typeparam name="TOut">TBD</typeparam>
         /// <typeparam name="TMat">TBD</typeparam>
+        /// <typeparam name="TMat2">TBD</typeparam>
         /// <param name="ops">TBD</param>
         /// <param name="sink">TBD</param>
         /// <returns>TBD</returns>
-        public static GraphDsl.Builder<TMat> To<TIn, TOut, TMat>(this GraphDsl.ForwardOps<TOut, TMat> ops, IGraph<SinkShape<TIn>, TMat> sink)
+        public static GraphDsl.Builder<TMat> To<TIn, TOut, TMat, TMat2>(this GraphDsl.ForwardOps<TOut, TMat> ops, IGraph<SinkShape<TIn>, TMat2> sink)
```

Comment:
Scala DSL does not care about the materializer value of the sink. We should do the same